### PR TITLE
add tesseract ocr script command

### PR DESCRIPTION
**Pull request by Dihgg**
## Description

Added a script that extract text from screenshot to clipboard using [Tesseract OCR](https://github.com/tesseract-ocr/tesseract)

## Type of change
- [x] New script command

## Screenshot
Screen Recording 2021-09-28 at 10 42 01 AM

## Dependencies / Requirements

Requires installation of [Tesseract OCR](https://github.com/tesseract-ocr/tesseract)
can be done by [brew install tesseract](https://formulae.brew.sh/formula/tesseract)

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by es183923** [comment]
You should use the `mptemp` command to generate a temporary file, instead of `/tmp/tmp.png`.

File: commands/productivity/tesseract/tesseract-ocr.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# @raycast.icon 🔍
+# @raycast.argument1 { "type": "text", "placeholder": "Language", "optional": true }
+# @raycast.packageName tesseract
+
+# Documentation:
+# @raycast.description Tesseract OCR
+# @raycast.author Diego Lopes
+# @raycast.authorURL https://github.com/Dihgg
+
+# echo "Hello World! Argument1 value: "$1""
```
Comment (es183923) [inline commands/productivity/tesseract/tesseract-ocr.sh:21]: This (line 21) is an unnecessary comment.

**Comment by es183923** [comment]
Other than that, I think it's good.

**Comment by dehesa** [comment]
Thank you @Dihgg!

**Pull request merged**